### ISSUE TITLE
As an User, I am able to fill, edit and view the form

### ISSUE THREAD

**Issue by Monal5031** [anitab-org/open-source-programs-web#10]
## Description
As an User, I am able to fill the form

## Mocks

See https://github.com/anitab-org/open-source-programs-web/wiki/User-Interface-Design for more insights on how it should look like

### Update [Required]
- [ ] Create form feedback page
- [ ] Add check for form deadline
- [ ] Connect API endpoint for creating form feedback
- [ ] Add validations on type of questions
- [ ] Create at least one test for each little feature: Fill form, Edit form and View form

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
2 days

**Comment by Aaishpra** [comment]
@isabelcosta i would like to work on this issue but since #8 and #9 are also connected to this issue can i also get assigned to #8 and #9 and work on all of them, together.

**Comment by isabelcosta** [comment]
@Aaishpra I agree, I understand these are big feature hard to breakdown. I am happy to assign those to you, however, can you get the change here done before https://github.com/anitab-org/open-source-programs-web/pull/57#issuecomment-766553783? its using css lib instead of custom CSS

**Comment by Aaishpra** [comment]
@isabelcosta Done with the changes, hope everything is good at #57 and i can get assigned here now.

**Comment by isabelcosta** [comment]
Will assign this and #9 #8 as part as the same issue. @Aaishpra

**Comment by isabelcosta** [comment]
@Aaishpra I edited the issue description so it has more accurate title + design links + writing tests acceptance criteria.

**Comment by Aaishpra** [comment]
> @Aaishpra I edited the issue description so it has more accurate title + design links + writing tests acceptance criteria.

Thanks for this 🎉

**Comment by Aaishpra** [comment]
Update : I am still working on this issue, and have completed some of my work, but due to poor health and hectic university schedule i will not be able to devote much time this week by working here, but i am still interested in completing this feature and would resume my work after being healthy.
Hope everyone is okay with that.

**Comment by isabelcosta** [comment]
@Aaishpra take care of your health ❤️ thank you for updating!
If you eventually prefer to deliver a small portion of this feel free to do this, I know this is a huge feature. You do not need to implement the whole thing.

**Comment by Aaishpra** [comment]
Thanks, @isabelcosta for understanding 🤗 let's see how much I can implement (when I get back to it again), I will update you if I change my mind about implementing just some features instead of the whole thing.

**Comment by SrijanShovit** [comment]
@isabelcosta Hey, is there something I can help with this issue?
I am trying to set up the project now.

**Comment by Navya1707** [comment]
@isabelcosta  @vatsalkul If this is still open, can I work on this issue?
